common/classify.py

```diff
@@ -9,6 +9,8 @@
 from sklearn import linear_model, cross_validation
 from sklearn.utils import shuffle
 from sklearn.externals import joblib
+from memtest import MemTracker
```

Comment:
`memtest` isn't part of the Python standard library right? Can you update `requirements.txt` to include this library so that future users will be able to install it?